cosmos/loadtest/scripts/address.py

```
@@ -0,0 +1,24 @@
+import bech32
+
+
+class Address:
+    def __init__(self, addr):
+        if type(addr) == str:
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)